src/robot/running/model.py

```diff
@@ -175,6 +176,34 @@ def run(self, context, run=True, templated=False):
                 raise ReturnFromKeyword(self.values)
 
 
+@Body.register
+class Continue(model.Return):
+    __slots__ = ['lineno']
+
+    def __init__(self, values=(), parent=None, lineno=None):
+        model.Return.__init__(self, values, parent)
```

Comment:
Neither `CONTINUE` nor `BREAK` accepts values in data (`RETURN` does) so model objects shouldn't have `values` arguments either.

Reply:
Removed following code blocked and directly  raised...  Is this correct or should I just remove values .... ?
with StatusReporter(self, ReturnResult(self.values), context, run):
            if run:
                raise ExitForLoop(self.values)